src/librustc_mir/transform/check_consts/qualifs.rs

```diff
@@ -103,11 +106,44 @@ impl Qualif for NeedsDrop {
         ty.needs_drop(cx.tcx, cx.param_env)
     }
 
-    fn in_adt_inherently(cx: &ConstCx<'_, 'tcx>, adt: &AdtDef) -> bool {
+    fn in_adt_inherently(cx: &ConstCx<'_, 'tcx>, adt: &'tcx AdtDef, _: SubstsRef<'tcx>) -> bool {
```

Comment:
nit: this line should have been in the previous commit, right?

(not that I'm a stickler for trying to ensure that every commit in a series is buildable; that ship sailed for us long ago...)